about_asserts.exs

```diff
@@ -29,10 +29,10 @@ defmodule AboutAsserts do
     is_1_equal_2? = fn -> assert 1 == 2 end
     is_1_greater_than_2? = fn -> assert 1 > 2 end
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
     assert_raise ExUnit.AssertionError, message, is_1_equal_2?
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
```

Comment:
I originally felt this way as well. I decided against it because the actual solution does not require interpolation. The style of these koans is "replace the `__?` with a solution". In this case the solution would be a string, and I feel like it'd be weird to expect something like `"Assertion with #{"=="} failed"`.

I don't feel incredibly strongly about it, so I might be convinced otherwise. What do you think?